packages/config-plugins/src/utils/history.ts

```diff
@@ -0,0 +1,36 @@
+import { ExpoConfig } from '@expo/config-types';
+
+import { ModPlatform } from '../Plugin.types';
+
+export type PluginHistoryItem = {
+  name: string;
+  version: string;
+  platform?: ModPlatform;
+};
+
+export function getHistoryItem(
+  config: Pick<ExpoConfig, 'extra'>,
+  name: string
+): PluginHistoryItem | null {
+  return config.extra?._pluginHistory?.[name] ?? null;
```

Comment:
`extra` is meant for users to add additional fields to make available to their app at runtime. we should move this to a new field like `_internal`

Reply:
or maybe `.expo` if this is intended to be persistent across runs